VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/linked-participant-panel-model.ts

```diff
@@ -40,8 +40,8 @@ export class LinkedParticipantPanelModel extends PanelModel {
         return this.participants.some(p => p.isWitness);
     }
 
-    get isWitnessReadyToJoin(): boolean {
-        return this.participants.every(p => p.isAvailable()) && this.participants.some(x => x.isWitness);
+    get isWitnessObserverReadyToJoin(): boolean {
```

Comment:
Can this be renamed to isWitnessOrQuickLinksObserverReadyToJoin as there is also the observer role.

Reply:
Will quick link participants also be added to this? Perhaps we should have a more generic name.

Reply:
@robertbennett1998 @FarronF  I will and have already refactored that in the story 7905. This was dealing with only observer, I kept it as it is.